StaggeredWaveToyX/test/standing/standing.par

```diff
@@ -16,9 +16,9 @@ CarpetX::ncells_x = 8
 CarpetX::ncells_y = 8
 CarpetX::ncells_z = 8
 
-CarpetX::periodic_x = yes
-CarpetX::periodic_y = yes
-CarpetX::periodic_z = yes
+Driver::periodic_x = yes
```

Comment:
Is a `Driver::periodic = yes` missing here?

Reply:
That's in the output directory. The parfile should be deleted there.